src/ADAL.PCL/Flows/AcquireTokenHandlerBase.cs

```diff
@@ -120,6 +117,8 @@ protected AcquireTokenHandlerBase(RequestData requestData)
 
                 if (this.LoadFromCache)
                 {
+                    PlatformPlugin.Logger.Verbose(CallState, string.Format(CultureInfo.InvariantCulture,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this formatting call and simply use the input string. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://iddp-sonarqube:9000/coding_rules#rule_key=csharpsquid%3AS3457)